CONTRIBUTING.md

```diff
@@ -1,6 +1,16 @@
 # Contributing
 
-## Workflow
+
+## Working With the Monorepo
```

Comment:
I don't have that strong of a preference, but I feel like for the detailed section it still makes sense to to a short overview of what submitting a change means and then go into details of the process

Reply:
My rational for it was to get someone up and running with Compass on their machine as quick as possible, and then making a change and contributing would be a second step. I renamed it to `Getting Started` - does that make a bit more sense?
I'm also cool with it both ways.